acrobat/blocks/prompt-card/prompt-card.js

```diff
@@ -37,6 +38,7 @@ async function createBlock(element, cfg) {
     'data-toast': cfg.toast,
     'daa-im': true,
     'daa-lh': 'Featured prompts | Executive summary',
```

Comment:
will this remain hard coded?

Reply:
Could we have a default behavior and an option for authors to switch in case is need it?